rkt/run.go

```diff
@@ -155,6 +156,7 @@ func init() {
 	cmdRun.Flags().StringVar(&flagHostname, "hostname", "", `pod's hostname. If empty, it will be "rkt-$PODUUID"`)
 	cmdRun.Flags().Var((*appsVolume)(&rktApps), "volume", "volumes to make available in the pod")
 	cmdRun.Flags().StringVar(&flagIPCMode, "ipc", "", `whether to stay in the host IPC namespace. Syntax: --ipc=[auto|private|parent]`)
+	cmdRun.Flags().StringVar(&flagKubeletLogDir, "kubelet-log-dir", "", "path to kubelet log dir")
```

Comment:
Why is this flag in `rkt run`? It should be for `rkt app sandbox`, right?

Reply:
This is a function `addAppFlags` which is used for almost every `app` command, and for sure it's used in `rkt app add`:

https://github.com/rkt/rkt/blob/master/rkt/app_add.go#L41